src/views/List.jsx

```diff
@@ -22,18 +23,31 @@ export function List({ data, listToken }) {
 		return <Navigate to="/" />;
 	}
 
+	//  const additemHandler = () => {
+	// 	return < Navigate to = '/add-item' />
+	//  }
+
```

Comment:
Also here, we can clean up the component and remove this code we don't need.

Reply:
All suggestions carried out. Thanks for your input